### PR TITLE
Refactor page controller

### PR DESCRIPTION
**Pull request by emodric**
Refactors page controller to use View API, including the response listeners and view parameters injection listeners.

### CODE REVIEW

**Comment by andrerom** [comment]
review ping @bdunogier 
*Note: travis is not running on this one because of the branch to branch PR*

**Comment by bdunogier** [comment]
I'm +1 with this.

@emodric you can add the base branch (ezp-24869-viewController_refactoring) to the branches block of .travis.yml to get it to run tests.

**Comment by emodric** [comment]
@bdunogier Is this okay? https://github.com/emodric/ezpublish-kernel/commit/06ccb1b20af206cee844757dc05ac0fb1f0ba74a

I don't see that the tests are running.

File: .travis.yml [hunk 1]
```diff
@@ -56,6 +56,7 @@ matrix:
 branches:
   only:
     - master
+    - ezp-24869-viewController_refactoring
```
Comment (bdunogier) [inline .travis.yml]: It's `ezp24869-viewController_refactoring`, no dash :-)

**Comment by emodric** [comment]
Aaaand they're off!

**Pull request closed without merging**